internal/ignition/ignition.go

```diff
@@ -421,6 +453,17 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte,
 		return err
 	}
 
+	// Create the 'ovirt-config.yaml' file required by the installer for
+	// the oVirt provider and append the OVIRT_CONFIG to the envirionment
+	// variables
+	if g.cluster.Platform.Type == models.PlatformTypeOvirt {
+		ovirtConfigPath, e := g.CreateOvirtConfig()
+		if e != nil {
+			return e
+		}
+		envVars = append(envVars, "OVIRT_CONFIG="+ovirtConfigPath)
+	}
```

Comment:
This part should be added as a function the provider code. 
All provider related stuff should be in the provider dir